specification/cognitiveservices/data-plane/TextAnalytics/stable/v3.0/TextAnalytics.json

```diff
@@ -0,0 +1,1190 @@
+{
+  "swagger":"2.0",
+  "info":{
+     "version":"v3.0",
+     "contact":{
+        "name":"Microsoft Cognitive Services",
+        "url":"https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/",
+        "email":"mlapi@microsoft.com"
+     },
+     "title":"Text Analytics Client",
```

Comment:
Can you confirm on the name ? I see the older swagger def, with `Text Analytics API (version number)` kind of pattern.

Reply:
Swagger title is always "Text Analytics Client". This will effect the client class name auto generated